app/Models/ContactInformation.php

```diff
@@ -38,7 +38,7 @@ public function contact(): BelongsTo
      */
     public function contactInformationType(): BelongsTo
     {
-        return $this->belongsTo(ContactInformationType::class, 'type_id');
```

Comment:
We need to update the `fillable` column to reflect this new key, otherwise it'll be a problem.